tracks/r/exercises/raindrops.md

```diff
@@ -0,0 +1,60 @@
+### Possible solution
+
+This solution is very intuitive and follows step-by-step the instructions in the exercise
+
+```r
+raindrops <- function(number) {
+  rain <- ""
+
+  if (number %% 3 == 0)
+    rain <- append(rain, "Pling")
+  
+  if (number %% 5 == 0)
+    rain <- append(rain, "Plang")
+  
+  if (number %% 7 == 0)
+    rain <- append(rain, "Plong")
+  
+  if (!(number %% 3 == 0 | number %% 5 == 0 | number %% 7 == 0)) # alternatively: if(rain == "")
+    return(as.character(number))
+  
+  paste(rain, collapse = "")
```

Comment:
I like this way of `paste`-ing in the very end. I think it would help make the 2nd & 3rd solution below clearer as well. Even at the cost of an intermediate `rain_sound` variable or something, can we un-nest L42 & 55?

Reply:
I will take a look at this and provide another solution soon -- thanks for your comments! they were really really helpful! :)

Reply:
What do you think of https://exercism.io/mentor/solutions/fe604ee08fee46d594999c5e7c3ba86f as another solution to highlight here?